tst/regression/CMakeLists.txt

```diff
@@ -90,18 +95,21 @@ include(${PROJECT_SOURCE_DIR}/cmake/TestSetup.cmake)
 list(LENGTH TEST_DIRS len)
 math(EXPR end_index "${len} - 1")
 
+message("List contnts ${EXTRA_TEST_LABELS}")
 foreach( index RANGE ${end_index})
   list(GET TEST_DIRS ${index} TEST_DIR)
   list(GET TEST_PROCS ${index} TEST_PROC)
   list(GET TEST_ARGS ${index} TEST_ARG)
-  setup_test(${TEST_DIR} ${TEST_ARG})
-  setup_test_coverage(${TEST_DIR} ${TEST_ARG})
-  setup_test_mpi(${TEST_PROC} ${TEST_DIR} ${TEST_ARG})
-  setup_test_mpi_coverage(${TEST_PROC} ${TEST_DIR} ${TEST_ARG})
+  list(GET EXTRA_TEST_LABELS ${index} TEST_LABELS )
+  message(STATUS "Label ${TEST_LABELS}")
+  setup_test(${TEST_DIR} ${TEST_ARG} "${TEST_LABELS}")
+  setup_test_coverage(${TEST_DIR} ${TEST_ARG} "${TEST_LABELS}")
```

Comment:
@Yurlungur See here `setup_test` is setting up the serial tests, TEST_PROC is not passed to this function. Maybe I should relabel to `setup_serial_test`.

Reply:
Nah, I think it's fine. I just read the diffs, not the whole files, so I got confused. You don't need to change anything here.